2/Allergies.py

```diff
@@ -0,0 +1,22 @@
+
+def determine_allergies(score):
+    """function 
+
+    Arguments:
+        score {integer} -- score of alergies
+
+    Returns:
+        list -- list of things that are person is alergic to
+    """
+    allergens = ["eggs", "peanuts", "shellfish", "strawberries",
+                 "tomatoes", "chocolate", "pollen", "cats"]
+
+    def filter_function(x): return score >> x & 1
```

Comment:
Avoid using one line instructions.

Reply:
I've used lambda( vs code formatted it into function definition o_0